app/scss/_colors.scss

```diff
@@ -0,0 +1,8 @@
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+// Colors
+//
+// A collection of the color variables for DIM.
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+
+$black: #000;
+$white: #fff;
```

Comment:
Ok, this could be a potential starting place for adding app-wide colors to this file. Plus it's easy to change "white" to an off-white color like `#f2f2f2` and it would globally change.

I guess since the project is already pretty set with it's styling this kind of abstraction might not be necessary. But I could see it being helpful with the other brand colors.

Would you prefer it get removed for now?